outputs.tf

```diff
@@ -4,27 +4,27 @@ output "database_name" {
 }
 
 output "master_username" {
-  value       = join("", aws_rds_cluster.default.*.master_username)
+  value       = coalesce(join("", aws_rds_cluster.primary.*.master_username), join("", aws_rds_cluster.secondary.*.master_username))
```

Comment:
I expect this will fail when the module is disabled. `coalesce` throws an error if all its arguments are empty.

Reply:
I face the same issue here

Reply:
this was fixed in https://github.com/cloudposse/terraform-aws-rds-cluster/pull/80 and is already in `master`